pkg/cvo/upgradeable.go

```diff
@@ -87,6 +88,7 @@ func (optr *Operator) setUpgradeableConditions() {
 			msgs = append(msgs, cond.Message)
 			cond.LastTransitionTime = now
 			conds = append(conds, *cond)
+			klog.V(2).Infof("Upgradeability Condition %s failed with Reason=%s(%s)", cond.Type, cond.Reason, cond.Message)
```

Comment:
nit: maybe add a space between the reason and message (`Reason=%s (%s)`)?  Or use a colon delimiter instead of parens (`Reason=%s: %s`)?  Or something?